### ISSUE TITLE
Calculation cost 

### ISSUE THREAD

**Issue by luobotaxinghu** [dongjun-Lee/transfer-learning-text-tf#1]
Could you provide details of GPU and how much time to run train.py and pre_train.py of auto-encoder and language_model ?

**Comment by dongjun-Lee** [comment]
I used Titan X, and it took about 2.5 hours for each epoch(64 batch_size * 8750 batches) for pre_train.py. Training auto-encoder and language model took almost the same time. For train.py (training text classification model), it took about 20 minutes for each epoch.

**Comment by luobotaxinghu** [comment]
Thank you. Do you record  loss  of  pre-train ?

**Comment by dongjun-Lee** [comment]
You're welcome! Yes, I did as [here](https://github.com/dongjun-Lee/transfer-learning-text-tf/blob/master/pre_train.py#L44-L45).
